src/DaPigGuy/PiggyCustomEnchants/CustomEnchantManager.php

```diff
@@ -193,6 +196,7 @@ public static function init(PiggyCustomEnchants $plugin)
         self::registerEnchantment(new ReviveEnchant($plugin, CustomEnchantIds::REVIVE));
         self::registerEnchantment(new SelfDestructEnchant($plugin, CustomEnchantIds::SELFDESTRUCT));
         self::registerEnchantment(new ShieldedEnchant($plugin, CustomEnchantIds::SHIELDED));
+        self::registerEnchantment(new ShrinkEnchant($plugin, CustomEnchantIds::SHRINK, CustomEnchant::RARITY_UNCOMMON));
```

Comment:
Uncommon Rarity can be removed, since it is the default value.

Reply:
No it isn't ;( 

https://github.com/DaPigGuy/PiggyCustomEnchants/blob/master/src/DaPigGuy/PiggyCustomEnchants/enchants/CustomEnchant.php#L63